test/features/bootstrap_init_join_remove.feature

```diff
@@ -0,0 +1,26 @@
+@bootstrap
+Feature: HA bootstrap process - init, join and remove
```

Comment:
I still don't like having tests depending in each other.
If the 1st test fails here, the rest will fail, and this doesn't mean that the things being tested in those are incorrect.

I would prefer to have totally independent tests.
Remember that the tests might be executable 1 by 1 and not only in CI

Reply:
Changed, every scenario with a `@clean` tag to make sure it have no dependency